homework/style.css

```diff
@@ -1,3 +1,53 @@
 .alert-error {
```

Comment:
On a wide-screen monitor your UI looks a bit stretched out. Perhaps set a max width and center the content horizontally?

![image](https://user-images.githubusercontent.com/2788771/65370909-4c7fef00-dc5e-11e9-8ca2-52f9b48f8ab5.png)